libmscore/layout.cpp

```diff
@@ -3602,38 +3602,24 @@ System* Score::collectSystem(LayoutContext& lc)
         bool doBreak = (system->measures().size() > 1) && ((minWidth + ww) > systemWidth);
         if (doBreak) {
             breakMeasure = lc.curMeasure;
-            if (lc.prevMeasure->noBreak() && system->measures().size() > 2) {
-                // TODO: why 2? should this really be the total size of the nobreak group?
-                // to make sure we don't remove everything?
```

Comment:
@MarcSabatella This was a typo, should have been 1. But, as you also pointed out, the `if/else` wasn't even necessary.